### PR TITLE
Update for Common Voice recipe & Multilingual training recipe

### LINKED ISSUE #1357: Sclite "Segmentation fault" encountered for Fr-Commonvoice
I recently found that the sclite for WER get "segmentation fault" when computing the decoded french trn files.

The error message is as follows:

`tools/espnet/utils/score_sclite.sh: line 121: 14394 Segmentation fault      sclite -r ${dir}/ref.trn trn -h ${dir}/hyp.trn trn -i rm -o all stdout > ${dir}/result.txt`

The two relating trn file is in the following links (for you to check the error):
https://drive.google.com/file/d/1vh7TbR_zHZI5yR3F3zTEufEIjID2VZS1/view?usp=sharing

### PR DESCRIPTION
**Pull request by ftshijt**
1. update RESULTS.md with the new setting (it seems the WERs drop much comparing to the original one, the reason should be the text-split. The previous train-dev-test split does not consider the text split, all the text in dev&test were in the train set.)
2. remove text with "{}" (only in French data
Fixes #1357 )
3. add multi-lingual recipe for #1459

### CODE REVIEW

**Comment by sw005320** [comment]
> 3. add multi-lingual recipe for #1459

This should not be under the commonvoice directory.
It's better to make a new recipe directory (e.g., `li35`).

**Comment by ftshijt** [comment]
> > 1. add multi-lingual recipe for #1459
> 
> This should not be under the commonvoice directory.
> It's better to make a new recipe directory (e.g., `li35`).

Got it, I would open up another PR regarding this

File: egs/li42/asr1/conf/lm.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+layer: 2
```
Comment (sw005320) [inline egs/li42/asr1/conf/lm.yaml]: Is it used?
(I don't think so).

**Comment by ftshijt** [comment]
> 1. can you upload all models?
> 2. add the li42 data descriptions to https://github.com/espnet/espnet/blob/master/egs/README.md

For the models, do I need to upload to your spack or simply provide it in my google drive?

**Comment by sw005320** [comment]
> For the models, do I need to upload to your spack or simply provide it in my google drive?

Either is fine.

File: egs/commonvoice/asr1/run.sh [hunk 2]
```diff
@@ -100,11 +100,9 @@ if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
     for x in ${train_set} ${recog_set}; do
       # Remove features with too long frames in training data
       max_len=3000
```
Comment (sw005320) [inline egs/commonvoice/asr1/run.sh]: Are `max_len=3000` and `max_chars=10000` appropriate?
Especially `max_chars=10000` looks very strange.

Comment (ftshijt) [reply egs/commonvoice/asr1/run.sh]: I mean to set a large threshold on character-level to follow the previous results (with only a 3000 frame threshold).

Comment (sw005320) [reply egs/commonvoice/asr1/run.sh]: why not using default one?
Some corpus includes very long sentences (by mistakes), and 10,000 will break training.

Comment (ftshijt) [reply egs/commonvoice/asr1/run.sh]: The default one should be enough, I will change it by next commit

**Comment by sw005320** [comment]
Thanks a lot!

**Pull request merged**